ocs_ci/ocs/perfresult.py

```diff
@@ -95,17 +95,16 @@ def dump_to_file(self):
 
     def es_write(self):
         """
-        Writing the results to the elastic-search server, if data can not be
-        written to the elasticsearch server, it will be written to JSON file
+        Writing the results to the elastic-search server, and to a JSON file
 
         """
 
         # Adding the results to the ES document / JSON file
```

Comment:
# Adding the results to the ES document and JSON file

Reply:
done.